settings.py

```diff
@@ -39,7 +37,7 @@
 UNIT_TEST_MODE = os.environ['SERVER_SOFTWARE'].startswith('test')
 
 
-APP_ID = app_identity.get_application_id()
+APP_ID = os.environ['APPLICATION_ID']
```

Comment:
I don't think that this quite works because
os.environ['APPLICATION_ID'] has a value with a "~" in it.
E.g., "s~cr-status-staging"

Also, while APPLICATION_ID seems to be present for python2 on appengine, it is not in the list of environment variables that are supposed to be there.  See:
https://cloud.google.com/appengine/docs/standard/python/runtime
and
https://cloud.google.com/appengine/docs/standard/python3/runtime

Both of those have GOOGLE_CLOUD_PROJECT, which seems to have just exactly the string that we need.  However, there is no GOOGLE_CLOUD_PROJECT when running locally.  So, you could use os.environ.get('GOOGLE_CLOUD_PROJECT', 'dev') which will return 'dev', which should be OK.

The condition on line 44 will fail because GOOGLE_COULD_PROJECT will never be 'testbed-test', so you could instead use UNIT_TEST_MODE for that condition.

Reply:
Done!